doc/docs/api/description.md

```diff
@@ -194,6 +198,10 @@ Object path: `/org/containers/hirte/node/$name`
 
     `Reload()` may be invoked to reload all unit files.
 
+  * `SetLogLevel(in s loglevel)`
```

Comment:
```suggestion
  * `SetLogLevel(in s log_level)`
```
I think for the parameters we use `snake_case`. Could you please change that for the others as well? (I know we are not 100% consistent here - so if you want to change the other `camelCases` to `snake_case` that would be awesome :))